plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -35,9 +35,16 @@
 # method can consistently format/word the failure report. These should not be
 # raised outside of this module.
 
-class ModelInstantiationError(Exception): pass
-class StoreFileError(Exception): pass
-class PackageMetadataError(Exception) : pass
+class ModelInstantiationError(Exception):
+    pass
+
+
+class StoreFileError(Exception):
+    pass
+
+
+class PackageMetadataError(Exception):
+    pass
```

Comment:
I'd recommend against including unrelated stylistic changes in this pull request, and including this as a separate pull request against master. Keeping this pull request/commit targeted to only the duplicate NEVRA problem keeps the pull request clean and simpler, which is also relevant to anyone who might want to backport this patch to an older version of Pulp.